### PR TITLE
fix(overthebox): resolve empty guides for some languages

### PR DESCRIPTION
**Pull request by Steffy29**
| Question         | Answer
| ---------------- | ---
| Branch?          | `develop`
| Bug fix?         | yes
| New feature?     | no
| Breaking change? | no
| Tickets          | Fix #UXCT-44, UXCT-288
| License          | BSD 3-Clause

## Description

Resolve empty guides for some languages
Refactoring the code for over the box parts

### CODE REVIEW

File: packages/manager/modules/overthebox/package.json [hunk 1]
```diff
@@ -37,6 +37,7 @@
     "ovh-angular-responsive-tabs": "^4.0.0",
     "ovh-api-services": "^9.44.0",
     "ovh-manager-webfont": "^1.2.0",
+    "ovh-ui-angular": "^3.16.3",
```
Comment (marie-j) [inline packages/manager/modules/overthebox/package.json:40]: Should be in a separate commit

**Comment by Steffy29** [comment]
@marie-j It's done.

File: packages/manager/modules/overthebox/src/overthebox/details/overTheBox-details.routing.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('overTheBoxes.overTheBox.details', {
+    url: '/details',
+    views: {
+      otbView: 'overTheBoxDetails',
+    },
+    resolve: {
+      serviceName: /* @ngInject */ ($transition$) =>
```
Comment (marie-j) [inline packages/manager/modules/overthebox/src/overthebox/details/overTheBox-details.routing.js:8]: Maybe it makes more sense to declare it in the parent state as it is where the param is declared and the siblings can benefit from this injection

File: packages/manager/modules/overthebox/src/overthebox/docs/translations/Messages_de_DE.json [hunk 3]
```diff
@@ -1,35 +1,35 @@
...
- "overTheBox_docs_vlan_configuration": "VLAN Konfiguration",
- "overTheBox_docs_rate_test_protocol": "Protocole de test de débit",
- "overTheBox_docs_modem_interface": "Créer une interface modem manuellement",
- "overTheBox_docs_multiwan_route": "Créer une route multiwan spécifique",
- "overTheBox_docs_firewall_rules": "Firewall-Regeln",
- "overTheBox_docs_remote_access": "Accès à distance",
- "overTheBox_docs_open_new_tab": "Neues Fenster"
-}
\ No newline at end of file
+  "overTheBox_docs_title": "Anleitungen",
```
Comment (marie-j) [inline packages/manager/modules/overthebox/src/overthebox/docs/translations/Messages_de_DE.json:2]: Due to prettier ?

File: packages/manager/modules/overthebox/src/overthebox/overTheBox.module.js [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+import '@ovh-ux/ng-ui-router-title';
+import '@ovh-ux/ng-tail-logs';
+import 'ovh-angular-responsive-tabs';
+import ngOvhUiConfirmModal from '@ovh-ux/ng-ovh-ui-confirm-modal';
+
+import 'ovh-ui-kit/dist/oui.css';
+import 'ovh-ui-kit/dist/oui-olt.css';
+import 'ovh-ui-kit-bs/dist/ovh-ui-kit-bs.min.css';
+import 'ovh-ui-kit-bs/dist/oui-bs3-olt.css';
+import 'ovh-manager-webfont/dist/css/ovh-font.css';
```
Comment (marie-j) [inline packages/manager/modules/overthebox/src/overthebox/overTheBox.module.js:17]: I'm wondering if those should instead be declared as peer Dependency and be imported once in the app to avoid importing multiple time the same css dist

File: packages/manager/modules/overthebox/src/overthebox/overTheBox.routing.js [hunk 5]
```diff
@@ -0,0 +1,39 @@
+const commonResolves = {
```
Comment (marie-j) [inline packages/manager/modules/overthebox/src/overthebox/overTheBox.routing.js:1]: There's a commonResolve for one state declaration ? Copy/Paste ?

File: packages/manager/modules/overthebox/src/overthebox/remote/overTheBox-remote.routing.js [hunk 6]
```diff
@@ -0,0 +1,12 @@
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('overTheBoxes.overTheBox.remote', {
+    url: '/remote',
+    views: {
+      otbView: 'overTheBoxRemote',
+    },
+    resolve: {
+      serviceName: /* @ngInject */ ($transition$) =>
```
Comment (marie-j) [inline packages/manager/modules/overthebox/src/overthebox/remote/overTheBox-remote.routing.js:8]: This could be declared in the parent state to avoid to duplicate the declaration, it will be easier to maintain if the param name change

File: packages/manager/modules/overthebox/src/overthebox/docs/translations/Messages_de_DE.json (hunk 3 above)
Comment (Steffy29) [reply packages/manager/modules/overthebox/src/overthebox/docs/translations/Messages_de_DE.json:2]: yes

**Comment by Steffy29** [comment]
For the moment I clode this PR, I need to find why it doesn't work on staging.
